Dockerfile

```diff
@@ -358,6 +358,9 @@ COPY --from=pkg-libressl-amd64 / /rootfs
 COPY --from=pkg-libseccomp-amd64 / /rootfs
 COPY --from=pkg-linux-firmware-amd64 /lib/firmware/bnx2 /rootfs/lib/firmware/bnx2
 COPY --from=pkg-linux-firmware-amd64 /lib/firmware/bnx2x /rootfs/lib/firmware/bnx2x
+COPY --from=pkg-linux-firmware-amd64 /lib/firmware/amd-ucode /rootfs/lib/firmware/amd-ucode
```

Comment:
you might try moving this to `initramfs-archive-amd64` stage with (for example):

```
COPY --from=pkg-linux-firmware-amd64 /lib/firmware/amd-ucode ./lib/firmware/amd-ucode
```

this will make it available in the proper place for